paasta_tools/utils.py

```diff
@@ -1402,6 +1390,16 @@ def get_branch_dict(self, service, branch):
         full_branch = '%s:paasta-%s' % (service, branch)
         return self.get(full_branch, {})
 
+    def get_branch_dict_v2(self, service, branch, deploy_group):
```

Comment:
No I forgot to include this (since nothing was using the v2 deployments until now).